IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -647,78 +662,98 @@ sub _build_subdir_named {
     return $path;
 }
 
-sub _build_platform {
+sub _build_contents_data {
```

Comment:
I've avoided some of the prior uses of basename here since in theory this should be able to identify play_files within subdirectories in the zipfile.